### ISSUE TITLE
Update for SD3.5? Specifically SD3.5 medium turbo

### ISSUE THREAD

**Issue by openSourcerer9000** [Jeff-LiangF/streamv2v#16]
This is a beautiful piece of software. IMO one of the best AI papers ever published but it seems it never quite caught on with the community. Would love to see it updated for the latest SD3.5, specifically support for this model:

https://huggingface.co/tensorart/stable-diffusion-3.5-medium-turbo

It would involve swapping out the unet and scheduler for the transformer and FlowMatchEulerDiscreteScheduler. I let AI agents take a stab at it (this lower level stuff is beyond my skillset), which runs without errors, but the results just don't look right. It seems to have hallucinated zero and one tensors here:         ```
# -----------------------------------------------------------------
        # Diffusion scheduler & helpers
        # -----------------------------------------------------------------
        self.scheduler = FlowMatchEulerDiscreteScheduler.from_config(pipe.scheduler.config)
        # alpha/β pre‑computations for `add_noise`
        self.alpha_prod_t_sqrt = torch.zeros(len(t_index_list), device=self.device)
        self.beta_prod_t_sqrt = torch.zeros_like(self.alpha_prod_t_sqrt)
        for i, t in enumerate(t_index_list):
            sigma_t = self.scheduler.sigmas[t]
            self.alpha_prod_t_sqrt[i] = torch.sqrt(1 - sigma_t**2)
            self.beta_prod_t_sqrt[i] = torch.sqrt(sigma_t**2)
        # Default Euler coefficients (can be overridden externally)
        self.c_out = torch.ones_like(self.alpha_prod_t_sqrt, device=self.device)
        self.c_skip = torch.zeros_like(self.alpha_prod_t_sqrt, device=self.device)
```

The branch is here if it's any help: https://github.com/openSourcerer9000/streamv2v/tree/SD3.5
